src/liberty/libertyProject.ts

```diff
@@ -203,6 +203,49 @@ export class ProjectProvider implements vscode.TreeDataProvider<LibertyProject>
 		statusMessage.dispose();
 	}
 
+	/**
+	 * Method asks the user to save the workspace first if it is untitled and the workspace contains morethan
+	 * one project. if not saved there are chances for the projects state not being saved and manually added
+	 * projects might not be visible in the liberty dashboard.
+	 */
+	public async checkUntitledWorkspaceAndSaveIt(): Promise<void> {
+		return new Promise((resolve) => {
+			try {
+				vscode.window.showInformationMessage(
+					'Please save the workspace first, as manually added projects to the dashboard may not persist in the next VS Code session if the workspace is not saved.',
```

Comment:
Suggestion for rewording is below, but please also move this message to https://github.com/OpenLiberty/liberty-tools-vscode/blob/main/src/locales/en.json.
```suggestion
					'Please save the workspace first. Projects that are manually added to the Liberty dashboard may not persist in the next VS Code session if the workspace is not saved before adding the project.',
```

Reply:
Message is corrected and is moved as suggested